software/grafana/README.md

```diff
@@ -0,0 +1,23 @@
+# Grafama
+
+### From the [Grafana](https://github.com/grafana/grafana) GitHub
+The open and composable observability and data visualization platform.
+Visualize metrics, logs, and traces from multiple sources like Prometheus, Loki, Elasticsearch, InfluxDB, Postgres and many more.
+
+### Initial credentials
+After the first start of Grafana, you can login using the following credentials:
+ - Username: admin
+ - Password: admin
+
+### Update support
+The egg _should_ keep the data and conf folder when reinstalling. Therefore a reinstallation with "latest" as the selected version can be used to update Grafana.
```

Comment:
It would keep the data unless the script extraction overwrites it. In that case, you can check if the folders exist, move them, update and then restore them. For example, mv conf conf.backup

Reply:
I added the logic to move the `conf `folder to a backup and back again in the installation script.
I did not add it for the `data` directory, as it isn't delivered by the tarball, I think we can leave that one just as is.